kymatio/scattering2d/frontend/__init__.py

```diff
@@ -0,0 +1,21 @@
+__all__ = ['Scattering2D']
+
+import logging
+
+class Scattering2D(object):
+    def __init__(self, *args, **kwargs):
+        if 'frontend' not in kwargs:
+            frontend='numpy'
+        else:
+            frontend=kwargs['frontend']
+            kwargs.pop('frontend')
+
+        try:
+            module = __import__(frontend + '_frontend', globals(), locals(), [], 1)
```

Comment:
Reminder to self to add way where frontend capitalization doesn't matter.

Reply:
please open an issue!

Reply:
you opened the issue